src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -50,10 +51,53 @@
     private static final String GROUP_COMPRESSION = "File Compression";
     private static final String DEFAULT_FILENAME_TEMPLATE = "{{topic}}-{{partition}}-{{start_offset}}";
 
+    private static final String GROUP_RETRY_BACKOFF_POLICY = "Retry backoff policy";
+    public static final String KAFKA_RETRY_BACKOFF_MS_CONFIG = "kafka.retry.backoff.ms";
+
     protected AivenCommonConfig(final ConfigDef definition, final Map<?, ?> originals) {
         super(definition, originals);
     }
 
+    protected static void addKafkaBackoffPolicy(final ConfigDef configDef) {
+        configDef.define(
+                KAFKA_RETRY_BACKOFF_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                null,
+                new ConfigDef.Validator() {
+
+                    static final long MAXIMUM_BACKOFF_POLICY = 86400000; // 24 hours
+
+                    @Override
+                    public void ensureValid(final String name, final Object value) {
+                        if (Objects.isNull(value)) {
+                            return;
+                        }
+                        assert value instanceof Long;
+                        final var longValue = (Long) value;
+                        if (longValue < 0) {
```

Comment:
How about Range Validator instead? https://kafka.apache.org/0110/javadoc/org/apache/kafka/common/config/ConfigDef.Range.html

Reply:
Good point the problem is the the error message is misleading

Reply:
How bad is it?

Reply:
I see this discussion is recurring in different PRs. I think we just need to create our own class in Commons